onward/app/controllers/OnwardControllers.scala

```diff
@@ -41,6 +41,7 @@ trait OnwardControllers {
   lazy val mostViewedGalleryController = wire[MostViewedGalleryController]
   lazy val videoEndSlateController = wire[VideoEndSlateController]
   lazy val richLinkController = wire[RichLinkController]
+  lazy val contentCardController = wire[ContentCardController]
```

Comment:
Again renaming this to "recommendedContent" or something similar would make it easier to understand what this was for